src/graph_notebook/notebooks/04-Machine-Learning/neptune_ml_utils.py

```diff
@@ -279,7 +289,7 @@ def __upload_to_s3(self, bucketname: str):
                     self.formatted_directory, file), bucket, f'{file_path}/{file}')
 
     def prepare_movielens_data(self, s3_bucket: str):
-        bucket_name = f'{s3_bucket}/neptune-formatted/movielens-100k'
+        bucket_name = f'{s3_bucket}/ml-formatted/movielens-100k'
```

Comment:
looks like this was changed b/c of a rename I did across the package. Reverting it